src/AppInstallerSharedLib/GroupPolicy.cpp

```diff
@@ -336,6 +354,15 @@ namespace AppInstaller::Settings
         json["Arg"] = Arg;
         json["Data"] = Data;
         json["Identifier"] = Identifier;
+        json["Explicit"] = Explicit;
+
+        // Trust level is represented as an array of trust level strings since there can be multiple flags set.
+        int i = 0;
+        for (std::string entry : TrustLevel)

```

Comment:
nit: There is a `for` loop syntax that is designed just for this...

Reply:
🤦